test/test_cuda_multigpu.py

```diff
@@ -1015,14 +1016,16 @@ def _test(device: Union[str, int, torch.device]):
             # Prevent PyTorch from reusing the allocated memory
             torch.cuda.empty_cache()
             torch.cuda.synchronize()
+            time.sleep(0.1)
             before_free_bytes, before_available_bytes = torch.cuda.mem_get_info(device)
-            # increasing to 8MB to force acquiring a new block and overcome blocksize differences across platforms
-            t = torch.randn(1024 * 1024 * 8, device=device)  # noqa: F841
+            # increasing to 512 MB to force acquiring a new block and overcome blocksize differences across platforms
+            t = torch.randn(1024 * 1024 * 512, device=device)  # noqa: F841
```

Comment:
Thinking a bit more about this, would this block size be queryable from the caching allocator? Previously I assumed that this is system dependent on the CUDA side but now I'm thinking it might be an artifact of the caching allocator setting? Might be worth running some standalone memory microbenchmarks to see what the default size of a block is on Jetson and why that's different from other platforms.

Also consider changing the allocation to `torch.empty` to speed things up as we don't check the results of the randn here.